string topological_map_name
string source_waypoint #waypoint where wait was performed
string target_waypoint #target waypoint in case door opens
bool opened
float64 wait_time